### DISCUSSION TITLE
Can not import "AddTrainValTestMask" class from "torch_geometric.transforms"

### DISCUSSION THREAD

**Discussion by zhihuat** [pyg-team/pytorch_geometric#3742] [Q&A] [answered]
Hi! Thanks for the fantastic tool first. 
I met a problem when importing "AddTrainValTestMask" from "torch_geometric.transforms".  The source code shows that it has been deleted in the latest version. Is there has any alternative Class to use? 
Thank you in advance!

**Comment by rusty1s** [comment, marked as answer]
It got renamed to `RandomNodeSplit` :)

**Comment by zhihuat** [comment]
Thank you very much! I replaced the "AddTrainValTestMask " with "RandomNodeSplit" It works great!
